project.clj

```diff
@@ -78,66 +92,72 @@
+                                   :source-map-timestamp true
+                                   :preloads [print.foo.preloads.devtools]
+                                   :closure-defines {goog.DEBUG true}
+                                   :external-config {:devtools/config {:features-to-install :all}}}}
+                       {:id "dev-server"
+                        :source-paths ["src/name_bazaar/server" "src/name_bazaar/shared"
+                                       "src/district0x/server" "src/district0x/shared"]
+                        :figwheel {:on-jsload "name-bazaar.server.dev/on-jsload"}
+                        :compiler {:main "name-bazaar.server.dev"
+                                   :output-to "dev-server/name-bazaar.js",
+                                   :output-dir "dev-server",
+                                   :target :nodejs,
+                                   :optimizations :none,
+                                   :closure-defines {goog.DEBUG true}
+                                   :source-map true}}
+                       {:id "server"
+                        :source-paths ["src"]
+                        :compiler {:main "name-bazaar.server.core"
+                                   :output-to "server/name-bazaar.js",
+                                   :output-dir "server",
+                                   :target :nodejs,
+                                   :optimizations :simple,
+                                   :source-map "server/name-bazaar.js.map"
+                                   :closure-defines {goog.DEBUG false}
+                                   :pretty-print false
+                                   :pseudo-names false}}
+                       {:id "min"
+                        :source-paths ["src"]
+                        :compiler {:main "name-bazaar.ui.core"
+                                   :output-to "resources/public/js/compiled/app.js"
+                                   :optimizations :advanced
+                                   :closure-defines {goog.DEBUG false}
+                                   :pretty-print false
+                                   :pseudo-names false}}
+                       {:id "dev-tests"
+                        :source-paths ["src/name_bazaar/server" "src/name_bazaar/shared"
+                                       "src/district0x/server" "src/district0x/shared"
+                                       "test/server"]
+                        :figwheel true
+                        :compiler {:main "server.run-tests"
+                                   :output-to "dev-tests/name-bazaar-tests.js",
+                                   :output-dir "dev-tests",
+                                   :target :nodejs,
+                                   :optimizations :none,
+                                   :verbose false
+                                   :source-map true}}
+                       {:id "browser-tests"
```

Comment:
Please remove browser tests for now. We may readd later when site gets more established